modules/opencc/1.1.8/MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+"Open Chinese Convert"
+
+module(
+    name = "opencc",
+    version = "1.1.8",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "darts-clone", version = "0.32")
+bazel_dep(name = "googletest", version = "1.15.0")
```

Comment:
Should googletest be a [dev_dependency](https://bazel.build/rules/lib/globals/module#parameters_1)?